File: programmers/위장.py
def solution(clothes):
    answer = {}
    for i in clothes :
        if i[1] in answer : answer[i[1]] += 1
        else : answer[i[1]] = 1
    
    ans = 1
    for i in answer :
        ans *= (answer[i]+1)
    
    return ans-1

File: programmers/위장.cpp
#include <string>
#include <vector>
#include <map>
using namespace std;

int solution(vector<vector<string>> clothes) {
    map<string,int> m;
    for(vector<string> i : clothes) m[i[1]]++;
    map<string,int> :: iterator iter;
    int answer = 1;
    for(iter = m.begin();iter!=m.end();iter++) answer *=(iter->second+1);
    return answer-1;
}
